src/app/page/cabinet/userDecks/user.decks.component.html

```diff
@@ -5,16 +5,124 @@
     <a [routerLink]="['/user/cabinet/manage/decks']" class="active">Manage decks</a>
   </div>
 
-  <div class="decks-list">
-    <div class="decks-list-item" *ngFor="let deck of decks; let i = index" (click)="onDeckClicked(deck, i)">
-      <span>{{deck.name}}</span>
-      <button class="btn btn-danger btn-sm"
-      style="float: right">-</button>
+  <div class="list-button-container">
+    <div class="decks-list">
+      <div class="decks-list-item" *ngFor="let deck of decks; let i = index" (click)="onDeckClicked(deck, i)">
+        <span>{{deck.name}}</span>
+        <button class="btn btn-danger btn-sm" data-toggle="modal" data-target="#delete-deck"
+                (click)="setSelectedDeck(deck)" style="float: right">-
```

Comment:
It is bad practice to use "style" in this place. Please remove in css file.

Reply:
Got it) It'll be ready in a moment!